src/analyze.js

```diff
@@ -566,7 +566,7 @@ module.exports = async function (id, code, job) {
               let pjsonPath = path.resolve(id, '../package.json');
               const rootPjson = path.resolve('/package.json');
               while (pjsonPath !== rootPjson && !existsSync(pjsonPath))
-                pjsonPath = path.resolve(pjsonPath, '../package.json');
+                pjsonPath = path.resolve(pjsonPath, '../../package.json');
```

Comment:
I'm surprised this works. Why would you jump up two directories each loop instead of one at a time?

Reply:
It's only going up one directory, the first `../` gets to the current directory, and the second one goes up to the next directory.

Reply:
This bug, illustrated:
```
> path.resolve('/foo/bla.txt', '..')
'/foo'
> path.resolve('/foo/bla.txt', '../bla.txt')
'/foo/bla.txt'
```